USAGE.md

```diff
@@ -123,8 +123,14 @@ The files must be served by a local web server.
 If you have Python installed, the easiest way is probably to run this
 one-line command from the repository:
 
+Python 2:
+
     python -m SimpleHTTPServer 8000
 
+Python 3:
+
+	python -m http.server 8000
```

Comment:
Can you fix the indentation here?  It should be 4 spaces.

Reply:
Sorry, my emacs was still set up to use tabs instead of spaces.